types/src/v0/impls/auction.rs

```diff
@@ -273,33 +263,38 @@ impl SolverAuctionResults {
 }
 
 impl HasUrls for SolverAuctionResults {
+    /// Get the urls to fetch bids from builders.
     fn urls(&self) -> Vec<Url> {
-        self.winning_bids()
-            .iter()
-            .map(|bid| bid.url())
+        // TODO can be ither cli or env
+        let url_s = env::var("ESPRESSO_GENERIC_BUILDER_URL").unwrap();
```

Comment:
Not sure how this is going to work in reality. I don't think we want to force use of the env var, but we don't have a place to hold the builder_url. Since HotShot is calling this, would it make more sense that hotshot knew about the builder_url and appends to the list. 

Otherwise we would have to add it as a field to `AuctionResults`, which means solver would need to know `generic_builder`.

Reply:
How does the sequencer normally get data from the configuration file?  I was initially thinking that this would be easy to get because the config would be readily available in the code.  Is that not the case?

Reply:
Hummm looking again... maybe the mistake is implementing `AuctionResultsProvider` on `SolverAuctionResults`.